cmd/up/up.go

```diff
@@ -904,11 +913,54 @@ func (up *upContext) shutdown() {
 		up.Forwarder.Stop()
 	}
 
+	if up.Dev.IsHybridModeEnabled() {
+		oktetoLog.Infof("stopping local process...")
+		shutdownHybridMode(up.hybridCommand.Process.Pid)
+	}
+
 	oktetoLog.Info("completed shutdown sequence")
 	up.ShutdownCompleted <- true
 
 }
 
+func shutdownHybridMode(pid int) {
+	if existProcess(pid) {
+		if err := syscall.Kill(-pid, syscall.SIGINT); err != nil {
+			oktetoLog.Warning("failed to stop gracefully local process related to command executed in hybrid mode: %s", err.Error())
```

Comment:
Could we output what the commands were and some CTA to terminate them manually? Might also need to inform that not terminating might cause spurious errors on next `okteto` usages.

Reply:
That's when the user specifies the command, but a common scenario is also to have an interactive terminal (using `bash` or whatever), what would we show in that case?